fixtures/webstudio-cloudlfare-template/vite.config.ts

```diff
@@ -0,0 +1,10 @@
+import {
+  vitePlugin as remix,
+  cloudflareDevProxyVitePlugin as remixCloudflareDevProxy,
+} from "@remix-run/dev";
+import { defineConfig } from "vite";
+import tsconfigPaths from "vite-tsconfig-paths";
```

Comment:
I use resolve.alias in other templates instead of this as we use very limited syntax and don't really need whole plugin.